src/components/TodoList/TodoList.tsx

```diff
@@ -1,4 +1,18 @@
 // Add the required types and props
-export const TodoList = () => (
-  <>TodoList markup</>
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+type TodoType = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<TodoType> = ({ todos }) => (
+  <>
+    <section className="TodoList">
+      {todos.map(todo => (
+        <TodoInfo key={todo.id} todo={todo} />
+      ))}
+    </section>
+  </>
```

Comment:
Same about react fragment 

```suggestion
  <section className="TodoList">
    {todos.map(todo => (
      <TodoInfo key={todo.id} todo={todo} />
    ))}
  </section>
```